### PR TITLE
Title was too short again, see desc 

### PR DESCRIPTION
**Pull request by Um-Brehon**
changed the transition of the introduction section on the home page and created a component that will be used to show all of our content section. Used it in the Cataract section. Sorry for the evil commit :(

### CODE REVIEW

File: src/components/Cataract/style.jsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import styled from 'styled-components';
+
+const SCataract = styled.section`
+  @media only screen and (min-width: 700px) {
+  }
+`;
+
+export default SCataract;
```
Comment (KeerhemFR) [inline src/components/Cataract/style.jsx]: Why empty style file?

**Review by KeerhemFR — CHANGES_REQUESTED** [review]
Saw one evil

**Review by KeerhemFR — APPROVED** [review]
See no more evil
Just be careful to change branch when you work and another component next time

File: src/components/Home/Introduction/index.jsx [hunk 2]
```diff
@@ -1,38 +1,103 @@
-import { NavLink } from 'react-router-dom';
+import React, { useState } from 'react';
 import ALJNB from '../../../assets/ALJNB.jpg';
```
Comment (Hacene010) [inline src/components/Home/Introduction/index.jsx:2]: Did you know that you can remove the ../../... with the last Loïc's config ?

**Review by Hacene010 — APPROVED** [review]

File: src/components/Home/Introduction/index.jsx (hunk 2 above)
Comment (Um-Brehon) [reply src/components/Home/Introduction/index.jsx:2]: I didn't know! kind of sick tho! :o

**Pull request merged**